functions.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+
+function checkSingleCardAndGetAllInfo(string $cardTitle): array {
+    $connectionString = 'mysql:host=db; dbname=mtg_cards';
+    $dbUsername = 'root';
+    $dbPassword = 'password';
+    $db = new PDO ($connectionString, $dbUsername, $dbPassword);
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $queryString = "SELECT * FROM `cards` WHERE `title` LIKE '$cardTitle'";
+    $query = $db->prepare($queryString);
+    $query->execute();
+    return $result = $query->fetch();
+}
+
+function createAllDisplayCards(array $results): string {
+    $outputString = '';
+
+
+    foreach ($results as $card) {
+        if (!is_array($card) ||
+            !array_key_exists('title', $card) ||
+            !array_key_exists('cardType', $card) ||
+            !array_key_exists('color', $card) ||
+            !array_key_exists('raritySet', $card)) {
+            return '';
+        }
+        $returnString = '';
+        $returnString .=  "<div class='display-card'>";
```

Comment:
you can just start off with
```php
        $returnString =  "<div class='display-card'>";
```

Reply:
also `returnString` is not a semantic name because it is not being returned - maybe `cardString`